.dockstore.yml

```diff
@@ -0,0 +1,17 @@
+version: 1.2
+workflows:
+  - name: intro-short
+    subclass: GXFORMAT2
```

Comment:
@coverbeck Can we make this Galaxy also as we're fixing that descriptor bug? GXFORMAT2 isn't appropriate for these workflows in this file either.

Reply:
@jmchilton Can we discuss at our CoFest meeting this afternoon? We felt this one made more sense than the TRS case, but sounds like not.

Reply:
Sounds good we can discuss it then, but these files are literally the older JSON files and not the newer format 2 YAML files. We can talk about it then.

Reply:
That sounds pretty compelling.

Reply:
I think at CoFest, I discussed the reverse actually. GALAXY can cover both the newer format and the older format. TRS doesn't distinguish between, say WDL draft-2 and v1 or CWL 1.0 and CWL v1.1. It is just WDL and CWL. 
TRS looks like it matches this expectation currently. 

We use GXFORMAT2 in a bunch of places internally which was my mistake. But we might be able to change the external representation for now without much pain?

Reply:
dockstore/dockstore#3686